CMakeLists.txt

```diff
@@ -810,14 +822,9 @@ foreach(sourcefile ${C_TEST_EXES})
     add_test(NAME ${exename} COMMAND ${exename}${ARTIFACT_SUFFIX})
     add_dependencies(check ${exename}${ARTIFACT_SUFFIX})
 endforeach(sourcefile ${C_TEST_EXES})
-add_subdirectory(tools)
+endif()
 
-# Installation and packaging for Linux
-if(NOT WIN32)
```

Comment:
> do you want to add something about CMake packages and Hunter?

If this library works on Windows and somebody want to use it, then there should be install procedure available, it's pretty obvious :) This is true even out of scope of creating CMake package (you have to run install even if no `*Config.cmake` will be installed) or Hunter (there is no dependencies).

I see few general CMake issues, can comment on them if anybody interested.